tests/conftest.py

```diff
@@ -2326,7 +2327,20 @@ def fixture_context_config(
         "pdc": {"binary_path": "/bin/dsmc"},
         "taxprofiler": {
             "binary_path": Path("path", "to", "bin", "nextflow").as_posix(),
+            "compute_env": "nf_tower_compute_env",
             "root": str(taxprofiler_dir),
```

Comment:
```suggestion
            "root":taxprofiler_dir.as_posix(),
```

Reply:
I got error when running the tests and I changed it back.

Reply:
Yes, you have to change the fixture to return a `Path`. An addition to that, this fixture appears duplicated in the `conftest`, so you should remove one of them

Reply:
Have you removed the `.as_posix()` from `return Path(taxprofiler_dir).absolute().as_posix()` in the fixture?

Reply:
My bad!